src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -232,6 +233,8 @@ def __init__(self, name, handler, description=None, table_transformer=None,
 
     @staticmethod
     def _should_load_description():
+        from azure.cli.core.application import APPLICATION
+
```

Comment:
Changes in this file shouldn't be required for this feature.
Are they for progress?